csrankings-2.csv

```diff
@@ -1608,6 +1608,7 @@ Florentin Wörgötter,University of Göttingen,http://www.uni-goettingen.de/en/5
 Florian Alt,Bundeswehr University Munich,https://www.unibw.de/usable-security-and-privacy/team/univ-prof-dr-florian-alt,wf3Muy8AAAAJ
 Florian Daniel,Politecnico di Milano,http://home.deib.polimi.it/daniel,wF6n9-0AAAAJ
 Florian Echtler,Bauhaus University Weimar,https://www.uni-weimar.de/de/medien/professuren/medieninformatik/mobile-media/people,sZssYagAAAAJ
+Florian "Floyd" Mueller,Monash University,https://research.monash.edu/en/persons/floyd-mueller

```

Comment:
Needs a single quote, not a double-quote, plus various aliases need to be added to `dblp-aliases.csv`.

https://dblp.org/pid/m/FlorianMueller-1.html

Reply:
Thanks quite a lot. Should I update and pull request again? Sorry for any inconvenience.